src/explore-education-statistics-admin/src/pages/release/content/components/ReleasePageTabHome.tsx

```diff
@@ -1,39 +1,205 @@
+        {
+          id: 'headlines-section',
+          text: 'Headline facts and figures',
+        },
+        ...contentSectionsItems,
+        contactUsNavItem,
+      ].filter(item => !!item),
+    [contentSectionsItems, hasSummarySection],
+  );
 
   return (
     <ReleasePageTabPanel tabKey="home" hidden={hidden}>
-      <ReleasePageLayout navItems={[]}>
-        <h2>{release.title}</h2>
-        <p>
-          Per melius aperiri eu. Et interesset philosophia vim, graece denique
-          intellegam duo at, te vix quot apeirian dignissim. Ei essent
-          percipitur nam, natum possit interpretaris sea ea. Cum assum adipisci
-          cotidieque ut, ut veri tollit duo. Erat idque volutpat mea ut, mel
-          nominati splendide vulputate ea. No ferri partem ceteros pro. Everti
-          volumus menandri at pro. Cum illud euripidis cu, mazim deterruisset ei
-          eum. Ex alia dolorem insolens per, malis clita laboramus duo ut,
-          ridens appareat philosophia ea quo.
-        </p>
-        <p>
-          Per melius aperiri eu. Et interesset philosophia vim, graece denique
-          intellegam duo at, te vix quot apeirian dignissim. Ei essent
-          percipitur nam, natum possit interpretaris sea ea. Cum assum adipisci
-          cotidieque ut, ut veri tollit duo. Erat idque volutpat mea ut, mel
-          nominati splendide vulputate ea. No ferri partem ceteros pro. Everti
-          volumus menandri at pro. Cum illud euripidis cu, mazim deterruisset ei
-          eum. Ex alia dolorem insolens per, malis clita laboramus duo ut,
-          ridens appareat philosophia ea quo.
-        </p>
+      <ReleasePageLayout navItems={navItems}>
+        {isMobileMedia && (
+          <section id="publication-release-intro-mobile">
+            <VisuallyHidden as="h2">Introduction</VisuallyHidden>
+
+            <p className="govuk-body-l">{publication.summary}</p>
+
+            <ReleaseSummaryBlockMobile
+              lastUpdated={updates[0]?.on}
+              releaseType={type}
+              renderProducerLink={
+                publishingOrganisations?.length ? (
+                  <span>
+                    {publishingOrganisations.map((org, index) => (
+                      <Fragment key={org.id}>
+                        {index > 0 && ' and '}
+                        <Link unvisited to={org.url}>
+                          {org.title}
+                        </Link>
+                      </Fragment>
+                    ))}
+                  </span>
+                ) : (
+                  <Link
+                    unvisited
+                    className="govuk-link--no-underline"
+                    to="https://www.gov.uk/government/organisations/department-for-education"
+                  >
+                    Department for Education
+                  </Link>
+                )
+              }
+              renderUpdatesLink={
+                updates.length > 1 ? (
+                  <Link to="#">
```

Comment:
Presumably this link will go somewhere eventually? It might be helpful to add a TODO with the jira ticket reference so it doesn't get missed.